t/app/controller/around.t

```diff
@@ -556,14 +556,184 @@ subtest 'check nearby lookup, cobrand custom distances' => sub {
             { lat => 51.752, lon => -1.256, mode => 'suggestions', contains => 0}, # 325m away
             { lat => 51.7485, lon => -1.256, mode => 'suggestions', contains => 0}, # 714m away
             { lat => 51.74, lon => -1.256, mode => 'suggestions', contains => 0}, # 1660m away
-
         ) {
-            $mech->get_ok('/around/nearby?latitude='.$test->{lat}.'&longitude='.$test->{lon}.'&filter_category=Pothole' . ( $test->{mode} ? '&mode='.$test->{mode} : ''));
+            $mech->get_ok( '/around/nearby?latitude='
+                    . $test->{lat}
+                    . '&longitude='
+                    . $test->{lon}
+                    . '&filter_category=Pothole'
+                    . ( $test->{mode} ? '&mode=' . $test->{mode} : '' )
+            );
             $mech->contains_or_lacks($test->{contains}, '[51.754926,-1.256179,"yellow",' . $p->id . ',"Open: Around page Test 1 for ' . $body->id . '","small",false]');
         }
     };
 };
 
+subtest 'check nearby lookup, cobrand custom distances per category' => sub {
```

Comment:
Not introduced by you, but these tests are making external requests to tilma (but not failing if internet not present, hence not spotted - I only noticed because I wondered why they were so slow). I've pushed a fix to master if you could rebase on that.